codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/Rust.kt

```diff
@@ -103,6 +112,7 @@ object TestWorkspace {
                 ),
             )
         cargoToml.writeText(workspaceToml)
+        cargoLock.copyTo(baseDir.resolve("Cargo.lock"), true)
```

Comment:
Why are we copying over the AWS SDK's `Cargo.lock` file? `TestWorkspace.subproject()` gets called by whitelabel integration tests too.